src/components/TodoList/TodoList.scss

```diff
@@ -0,0 +1,45 @@
+.TodoList {
+  width: 100%;
+
+  &__interactive {
+    padding: 10px;
+    display: flex;
+    justify-content: space-between;
+    color: #4d457b;
+    font-weight: bold;
+
+    &-input {
+      padding: 5px 10px;
+      border-radius: 5px;
+      border: 2px solid #4d457b;
+    }
+  }
+
+  &__user-button--selected {
+    background-color: #4d457b;
```

Comment:
you're repeting yourself with this color alot, save it to a variable

Reply:
it doesn't mean that you shouldn't try to improve it more, always aim for something better)